README.md

```diff
@@ -134,7 +136,8 @@ test('should render mock graphql responses', async () => {
 
   expect(getByText(/Loading/)).toBeDefined()
 
-  await new Promise(resolve => setTimeout(resolve, 0))
+  await flushPromises()
+  jest.runAllTimers()
```

Comment:
Doesnt `wait` fulfill this need already?

https://testing-library.com/docs/dom-testing-library/api-async#wait

Reply:
`await wait(() =>{})`

Reply:
not really, because the use case I wanted this for is to wait for the graphql mock promise to be resolved, and using this with `jest.runAllTimers` makes the promise resolution instant.

> `await wait(() => {})`

the bad side of doing this is to have to wait at least 5 seconds on every test that has a graphql call